releasenotes/notes/chdir-to-writable-dir-for-in-toto-68a834c59e9986ff.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    For in-toto, run pip in a directory writable by the Agent, so that we can
+    switch back and forth from temporary working directories.
```

Comment:
it'd be better to word this in a user-facing way, i.e. describe what the change fixes instead of what it allows us to do internally.